private_sharing/models.py

```diff
@@ -655,7 +665,7 @@ class ProjectDataFile(DataFile):
         related_name="parent_project_data_file",
         on_delete=models.CASCADE,
     )
-
+    categories = models.ManyToManyField(Ontology, related_name="datafile_categories")
```

Comment:
Regarding "categories" and "Ontology": I think "DataType" everywhere would be more consistent and clear.

- the model "Ontology" could be "DataType"
- the field "categories" could be "datatypes"